docs/self-managed/platform-deployment/helm-kubernetes/upgrade.md

```diff
@@ -93,6 +93,15 @@ For more details on the Keycloak upgrade path, you can also read the [Bitnami Ke
 
 ## Version update instructions
 
+### v8.4.0
```

Comment:
Is there any other related documentation we should be noting this in given the breaking changes?

Reply:
I will work on this just after updating the parts related to the Helm chart versioning change.

Reply:
Done :heavy_check_mark: